docs/reference/remap.cue

```diff
@@ -49,7 +49,7 @@ remap: {
 
 			arguments: [...#Argument] // Allow for empty list
 			return: [#RemapReturnTypes, ...#RemapReturnTypes]
-			category:    "Coerce" | "Encode" | "Enumerate" | "Event" | "Hash" | "IP" | "Map" | "Number" | "Parse" | "Random" | "String" | "Test" | "Timestamp"
+			category:    "Check" | "Coerce" | "Encode" | "Enumerate" | "Event" | "Hash" | "IP" | "Map" | "Number" | "Parse" | "Random" | "String" | "Test" | "Timestamp"
```

Comment:
Since we're adding the new category in this PR, should we update existing docs that should fall under this one as well? Or will you do that in a follow-up PR?

Reply:
I'll do a follow-up PR that generally re-assesses the categories. They've been kind of ad hoc thus far, and that's something we can address all at once.